h/static/styles/partials-v2/_elements.scss

```diff
@@ -23,3 +23,9 @@ input[type=text],
 textarea {
   appearance: none;
 }
+
+// Hide contents of <template> elements in browsers that do not support
+// this element natively (eg. IE).
+template {
```

Comment:
`<template>` elements are inert and do not render anything in browsers that support it natively. In browsers that don't, `<template>` behaves like an ordinary unrecognized Element.